ModList/UI/ModListFlowCoordinator.cs

```diff
@@ -1,87 +1,120 @@
-﻿using BeatSaberMarkupLanguage;
-using HMUI;
-using System;
-using System.Collections.Generic;
+﻿using System;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using BeatSaberMarkupLanguage;
+using HMUI;
+using IPA.Loader;
+using IPA.ModList.BeatSaber.Models;
+using IPA.ModList.BeatSaber.UI.ViewControllers;
+using SiraUtil.Tools;
+using Zenject;
 
 namespace IPA.ModList.BeatSaber.UI
 {
-    internal class ModListFlowCoordinator : FlowCoordinator
+    public class ModListFlowCoordinator : FlowCoordinator
```

Comment:
Why should the flow coordinator be public?

Reply:
Accidental change